carma/launch/plugins.launch.py

```diff
@@ -479,6 +479,9 @@ def generate_launch_description():
                     {'--log-level' : GetLogLevel('trajectory_follower_wrapper', env_log_levels) }
                 ],
                 remappings = [
+                    ("input/current_kinematic_state", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''),"/input/current_kinematic_state"]),
+                    ("input/reference_trajectory", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''),"/input/reference_trajectory"]),
+                    ("output/control_cmd" , [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''),"/output/control_cmd"]),
```

Comment:
If we follow comments on the guidance.launch.py, this would change to match as well